framework/contracts/native/version-control/src/commands.rs

```diff
@@ -357,12 +370,40 @@ pub fn claim_namespace(
         });
     }
 
+    let fee_msg = claim_namespace_internal(
```

Comment:
line 366 the account factory should no longer have permissions to claim namespaces through this function.

Reply:
Great nit, thanks!